webservice/src/main/java/dk/dbc/laesekompas/suggester/webservice/SolrBean.java

```diff
@@ -0,0 +1,87 @@
+package dk.dbc.laesekompas.suggester.webservice;
+/*
+ * Copyright (C) 2019 DBC A/S (http://dbc.dk/)
+ *
+ * This is part of suggester-laesekompas-webservice
+ *
+ * suggester-laesekompas-webservice is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * suggester-laesekompas-webservice is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * File created: 25/01/2021
+ */
+
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.ejb.Singleton;
+import javax.inject.Inject;
+
+@Singleton
+public class SolrBean {
```

Comment:
`@Lock(READ)` vil jeg gerne have på. Godt nok et metoderne så hurtige at det i praksis ikke er et problem, men bare for at få det ind på ryggraden,